cloudformation/hello-world/stack.yaml

```diff
@@ -0,0 +1,6 @@
+Resources:
+  S3Bucket:
+    Type: 'AWS::S3::Bucket'
+    DeletionPolicy: Retain
+    Properties:
+      BucketName: hello-world-bucket-123123123
```

Comment:
will it just run over existing bucket? since its a constant name...

Reply:
please use a dynamic name, these tests often run multiple times in parallel or fail to destroy ... we need to have unique names otherwise we get a lot of flakiness

Reply:
https://github.com/env0/templates/blob/chore-cf-template/aws/cf-hello-world/stack.yaml

Reply:
just used @chpl example take from AWS examples 😄